### PR TITLE
feat: OG card for drop page

### PR DESCRIPTION
**Pull request by Jarsen136**
Close https://github.com/kodadot/nft-gallery/issues/8316

preview: https://www.opengraph.xyz/url/https%3A%2F%2Fog-drop-image.workers-ogi.pages.dev%2Fahp%2Fdrops%2Fswirls

image

### CODE REVIEW

File: ssr-opengraph/src/index.tsx [hunk 1]
```diff
@@ -41,7 +41,12 @@ app.on(['GET', 'HEAD'], '/:chain/:type/:id/*', async (c) => {
   const type = c.req.param('type');
 
   if (chains.includes(chain)) {
-    if (type === 'gallery' || type === 'detail' || type === 'collection') {
+    if (
+      type === 'gallery' ||
+      type === 'detail' ||
+      type === 'collection' ||
+      type === 'drops'
+    ) {
```
Comment (vikiival) [inline ssr-opengraph/src/index.tsx:49]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/includes

**Review by vikiival — COMMENTED** [review]
Otherwise oki

**Comment by preschian** [comment]
> Now I'm faced with a tricky problem. I have to set `ssr: false`

Checking, afaik ssr is required for og-image

**Comment by preschian** [comment]
Oooh, I think that's because we tried to call workers to workers. I also face a similar issue: https://github.com/kodadot/workers/pull/201#issue-1993050599. Another workaround is to deploy `ogi` to Pages instead of Workers. So we can use our workers.

Just to let you know, I will try to integrate `ogi` into Pages in our CF accounts 🙏. cc @yangwao @vikiival

**Comment by preschian** [comment]
I just merged #214. Let's try it by removing `ssr: false` and changing opengraph domain https://github.com/kodadot/workers/blob/main/ssr-opengraph/src/utils.ts#L108 to https://workers-ogi.pages.dev/

no need to deploy `ogi` to workers, just need `ssr-opengraph` to workers

**Comment by Jarsen136** [comment]
Let' merge it.

**Pull request merged**